src/js/containers/search/filters/AgencyListContainer.jsx

```diff
@@ -43,7 +43,8 @@ export default class AgencyListContainer extends React.Component {
         let noResults = false;
 
         // Format results of search for use in Autocomplete component
-        if (results && results.length > 0) {
+        if (results) {
+            console.log('results: ', results);
```

Comment:
console log to remove

Reply:
Dang console logs, they're so sneaky. I've removed it.